doc/river.1.scd

```diff
@@ -0,0 +1,35 @@
+RIVER(1) "github.com/ifreund/river" "General Commands Manual"
+# NAME
+
+river - dynamic tiling Wayland compositor
+
+# SYNOPSIS
+
+*river* [*-c* _shell_command_]
+
+# DESCRIPTION
+
+*river* is a dynamic tiling Wayland compositor inspired by dwm and
+bspwm based on wlroots and written in Zig.
+
+# OPTIONS
+
+*-c* _shell_command_
```

Comment:
I think `*-c <shell comand>*` is more idiomatic for scdoc.

Reply:
I also see angle brackets used for arguments a lot. Man pages use italics by convention, though.